docs/topics/developers.md

```diff
@@ -83,13 +83,21 @@ Docker daemon:
 $ eval $(minikube docker-env)
 ```
 
-Running `make docker-build docker-push` will push the Brigade images to the Minikube Docker
+Running `VERSION=latest make docker-build docker-push` will push the Brigade images to the Minikube Docker
```

Comment:
Ah, nice. I had been blindly building brigade components without the `VERSION=latest` and setting `controller.tag`, `api.tag`, `worker.tag` and so on while installing the brigade 😅